Makefile

```diff
@@ -1,7 +1,11 @@
 PATH := node_modules/.bin/:$(PATH)
 
+ifdef NODEMON
+start: start-nodemon
+else
 start:
-	node server.js
+	@node server.js
+endif
```

Comment:
Maybe this leaked in by accident from #4614?

Reply:
The perils of `git -a`...

Reply:
I know 😄